### DISCUSSION TITLE
PHP and Laravel support

### DISCUSSION THREAD

**Discussion by bastien-phi** [hotmeteor/spectator#159] [Ideas]
Currently, Spectator supports Laravel 8, 9 and 10 and PHP from 7.4 to 8.3. Laravel 11 will land in a few month and it would be great to have support for it from day 1.

IMHO, having that much support will complicate the maintenance and will prevent us to use last improvement from both the framework and the language.

I suggest we limit the support to the supported versions of Laravel and their supported PHP versions : 
https://laravel.com/docs/10.x/releases#support-policy

![image](https://github.com/hotmeteor/spectator/assets/10199039/3ae17762-968b-44ee-b96d-9c03135ba42e)

Any opinion about that ?

**Comment by hotmeteor** [comment]
IMO we tag a V2 that's Laravel 10/11+ and PHP ^8.1

V1 stays, but gets no more updates.

**Reply by bastien-phi** [reply]
Let's do that !

**Reply by hotmeteor** [reply]
OK I'll start a branch for that.
